jsons/Policies.json

```diff
@@ -104,14 +104,14 @@
 					"[+15]% Production when constructing [Library] buildings [in all cities]",
 					"[+15]% Production when constructing [Colosseum] buildings [in all cities]",
 					"[+15]% Production when constructing [Opera House] buildings [in all cities]"]
-			}
-		]
+	}
+	]
 	},
 	{
 		"name": "Honor",
 		"era": "Ancient era",
-		"uniques": ["[+33]% Strength vs [Barbarians]","Earn [33]% of killed [Military] unit's [Strength] as [Culture]",
-			"Notified of new Barbarian encampments","Unlocks building the Temple of Artemis"],
+		"effect": "+25% bonus vs Barbarians; gain Culture when you kill a barbarian unit",
+		"uniques": ["+25% bonus vs Barbarians","Gain Culture when you kill a barbarian unit","Notified of new Barbarian encampments","Unlocks building the Temple of Artemis"],
```

Comment:
Not sure what happened here

Reply:
I am not sure as well

Reply:
At the very least: "effect" was only ever a thing for promotions and has been deprectaed for months, and all uniques here are also _very_ outdated

Reply:
If you want effects similar to the old uniques, you'd need:
```
"uniques": ["[+25]% Strength <vs [Barbarian] units>", "Earn [33]% of killed [Military] unit's [Strength] as [Culture]", "Notified of new Barbarian encampments", "Unlocks building the Temple of Aretmis"
```
Do note that this last unique doesn't actually do anything.